mozilla_django_oidc/auth.py

```diff
@@ -247,15 +255,6 @@ def get_or_create_user(self, access_token, id_token, verified_id):
         """Returns a User instance if 1 user is found. Creates a user if not found
         and configured to do so. Returns nothing if multiple users are matched."""
 
-        session = self.request.session
-
-        if import_from_settings('OIDC_STORE_ACCESS_TOKEN', False):
```

Comment:
Ideally i would prefer we move storing tokens to another method since it's quite self contained.

Reply:
Sounds reasonable, will do that.